src/lib/middleware/local_assets.coffee

```diff
@@ -0,0 +1,9 @@
+#
+# Serves assets from CDN_URL pointing to /public/assets
+#
+
+{ CDN_URL } = require '../../config'
+
+module.exports = (req, res, next) ->
+  res.locals.asset = (filename) -> return filename # no-op for now
```

Comment:
This might presumably be `CDN_URL + filename` or something.

Reply:
Actually it will probably also need to add the `.jgz` or `.cgz` extension as well.